flink-runtime/src/test/java/org/apache/flink/runtime/state/TaskExecutorLocalStateStoresManagerTest.java

```diff
@@ -205,17 +205,20 @@ private void checkRootDirsClean(File[] rootDirs) {
 	private TaskManagerServicesConfiguration createTaskManagerServiceConfiguration(
 			Configuration config) throws IOException {
 		return TaskManagerServicesConfiguration.fromConfiguration(
-			config, MEM_SIZE_PARAM, InetAddress.getLocalHost(), true);
+			config, InetAddress.getLocalHost());
```

Comment:
separate line for every parameter or we do not need break line here because it might not be long after removing above parameters.